seeds/_sector/dex/ramses_arbitrum_base_trades_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+arbitrum,ramses,1,2024-01-03,0x58114cc96d4b774f9acb6fc8e5c2cdb430466a455326125742608e6c145a3f30,8,0x178412e79c25968a32e89b11f63b33f733770c2a,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,166785045,50050294431056362,50000000000000000
+arbitrum,ramses,2,2024-01-03,0xca091c81a67166577ebe44515f07c2fa32159b51089237758118da51494b68ed,3,0xaaa6c1e32c55a7bfa8066a6fae9b42650f262418,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,166784031,10136918586602086743997,300000000000000000
```

Comment:
Hi @echovl, please verify this line of seed:
https://dune.com/queries/3330219

Reply:
Seed was failing for v2 because we had a wrong `RamsesV2Factory` indexed, I've submitted the correct `RamsesV2Factory`, tests are working now.